ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/util/PDEModelUtility.java

```diff
@@ -412,9 +412,11 @@ private static void modifyEditorModel(final ModelModification mod, final PDEForm
 				for (IFile file : files) {
 					if (file == null)
 						continue;
-					InputContext con = editor.getContextManager().findContext(file);
-					if (con != null)
-						con.flushEditorInput();
+					if (editor.getContextManager() != null){
+						InputContext con = editor.getContextManager().findContext(file);
+						if (con != null)
+							con.flushEditorInput();
+					}
```

Comment:
The method `editor.getContextManager()` should only be called once and the null-check should be done on the saved result of that only call. Otherwise it might happen that the first call returns a non-null value while the second call returns null again:
```suggestion
					InputContextManager manager = editor.getContextManager();
					if (manager != null){
						InputContext con = manager.findContext(file);
						if (con != null) {
							con.flushEditorInput();
						}
					}
```